projects/node-api/src/Client.ts

```diff
@@ -73,14 +73,18 @@ export class Client {
     constructor (
         username: string, password: string,
         protocol = 'https', host = 'hub.graphistry.com',
-        clientProtocolHostname?: string
+        clientProtocolHostname?: string,
+        previousClient?: Client
     ) {
         this.username = username;
-        this._password = password;
+        this.password = password;
         this.protocol = protocol;
```

Comment:
@dess890 we can probably go back to `private _password` just as `js-upload-api` is doing it, vs `public readonly password`, right?